src/hotspot/share/gc/shared/workgroup.hpp

```diff
@@ -327,17 +327,18 @@ class SubTasksDone: public CHeapObj<mtInternal> {
   // to be within the range of "this".
   bool try_claim_task(uint t);
 
-  // The calling thread asserts that it has attempted to claim all the
-  // tasks that it will try to claim.  Every thread in the parallel task
-  // must execute this.  (When the last thread does so, the task array is
-  // cleared.)
+  // The calling thread asserts that it has attempted to claim all the tasks
+  // that it will try to claim.  Tasks that is meant to be skipped must be
+  // explicitly passed as extra arguments using the variadic version below.
```

Comment:
I would drop "using the variadic version below".  I think of this as a function that takes some optional "these are expected to be skipped" task designators, and the non-variadic overload is just an implementation detail to handle the base case of there being none of those.